LetsGame/js/scriptssp.js

```diff
@@ -0,0 +1,72 @@
+/* Schere Stein Papier*/
+let _userPoint = 0; //Benutzerpunkte
+let _pcPoint = 0; //Pc-Punkte
+let _roundsToPlay = 0; //Rundenzähler
+
+function playSSP(humanPlayer){
+
+    if (_roundsToPlay == 0){ //Erste Runde - hole Max.Runden
+        _roundsToPlay = document.getElementById("counterInput").value;
+    }
+    let computerPlayer = Math.floor(Math.random()*2); //Pc - wählt Schere, Stein oder Papier
+    let winner = 0;
+    
+    //Zeigt Wahl vom Benutzer an
+    if (humanPlayer == 0){
+        userImg.setAttribute("src", "img/ssp/schere.gif");
+    }
+    else if (humanPlayer == 1){
+        userImg.setAttribute("src", "img/ssp/stein.gif");
+    }
+    else {
+        userImg.setAttribute("src", "img/ssp/papier.gif");
+    }
+    //Zeigt Wahl vom PC an
+    if (computerPlayer == 0){
+        pcImg.setAttribute("src", "img/ssp/schere.gif");
+    }
+    if (computerPlayer == 1){
+        pcImg.setAttribute("src", "img/ssp/stein.gif");
+    }
+    if (computerPlayer == 2){
+        pcImg.setAttribute("src", "img/ssp/papier.gif");
+    }   
+
+    //Pc und Benutzer haben verschiedene Symbole
+    if (humanPlayer != computerPlayer){
+        winner = 2;
+        if ((humanPlayer == 0) && (computerPlayer == 2)){//Schere schlägt Papier
+            winner = 1;
+        }
+        if ((humanPlayer == 1) && (computerPlayer == 0)){//Stein schlägt Schere
+            winner = 1;
+        }
+        if ((humanPlayer == 2) && (computerPlayer == 1)){//Papier schlägt Stein
+            winner = 1;
+        }

```

Comment:
Arbeite immer mit **if else**!
Du gehst mir in jede Bedingung rein. Wenn humanPlayer == 0 dann kann humanPlayer nicht mehr 1 sein.